internal/project/framework/internals/sentry/sentry.go

```diff
@@ -1,11 +1,12 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package sentry
 
 import (
 	"errors"
 	"reflect"
 
-	"{{ .PkgPath }}/framework/internals/global"
+	/**#bean*/
+	"demo/framework/internals/global" /*#bean.replace("{{ .PkgPath }}/framework/internals/global")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines